modules/uhttp/client/client_filters_test.go

```diff
@@ -74,7 +74,7 @@ func TestExecutionChainFiltersError(t *testing.T) {
 }
 
 func withOpentracingSetup(t *testing.T, registerFunc auth.RegisterFunc, fn func(tracer opentracing.Tracer)) {
-	tracer, closer, err := tracing.InitGlobalTracer(&jconfig.Configuration{}, "Test", ulog.NopLogger, metrics.NopCachedStatsReporter)
+	tracer, closer, err := tracing.InitGlobalTracer(&jconfig.Configuration{}, "Test", zap.New(nil), metrics.NopCachedStatsReporter)
```

Comment:
Consider `zap.Nop` instead of `zap.New(nil)` for NopCore. 
I actually had to pull up zap to see what it meant to pass in nil